internal/setup/setup.go

```diff
@@ -638,6 +677,17 @@ func parsePackage(baseDir, pkgName, pkgPath string, data []byte) (*Package, erro
 	return &pkg, err
 }
 
+func validateGeneratePath(path string) error {
+	if !strings.HasSuffix(path, "/**") {
+		return fmt.Errorf("has invalid path %s: does not end with /**", path)
```

Comment:
Sorry for not rising this earlier but the error messages here should be somewhat self-contained so `has...` does not work. Maybe you can do `does not end with /**` and in the other one `slice %s_%s has invalid generate path: %s` or something along those lines.

Reply:
Done, thanks! ad54c29